src/Entity/Project/Cca.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace App\Entity\Project;
+
+use App\Entity\Personne\Prospect;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="App\Repository\Project\CcaRepository")
+ * @UniqueEntity("nom")
+ */
+class Cca extends DocType
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="date")
+     */
+    private $dateFin;
+
+    /**
+     * @ORM\OneToMany(targetEntity=Etude::class, mappedBy="cca")
+     */
+    private $etudes;
+
+    /**
+     * @Assert\Valid()
+     * @ORM\ManyToOne(targetEntity=Prospect::class, cascade={"persist"})
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $prospect;
+
+    /************************
+     *   OTHERS DEFINITIONS
+     ************************/
+
+    /**
+     * @var bool
+     */
+    private $knownProspect = false;
+
+    /**
+     * @var Prospect
+     */
+    private $newProspect;
+
+    /**
+     * @ORM\Column(type="string", length=50, nullable=false, unique=true)
+     */
+    private $nom;
+
+    /**
+     * @ORM\OneToMany(targetEntity=Ce::class, mappedBy="cca")
+     */
+    private $bdcs;
+
+    public function __toString()
+    {
+        return $this->getNom() . ' - ' . $this->dateFin->format('d/m/Y');
+    }
+
+    public function __construct()
+    {
+        $this->etudes = new ArrayCollection();
+        $this->bdcs = new ArrayCollection();
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getDateFin(): ?\DateTimeInterface
+    {
+        return $this->dateFin;
+    }
+
+    public function setDateFin(\DateTimeInterface $dateFin): self
+    {
+        $this->dateFin = $dateFin;
+
+        return $this;
+    }
+
+    /**
+     * @return Collection|Etude[]
+     */
+    public function getEtude(): Collection
```

Comment:
erreur dans le nommage de la methode. Retourne plusieurs études, donc 

```suggestion
    public function getEtudes(): Collection
```

Attention aux effets de bords de ce renommage dans les templates.